src/client/components/NavbarDropdown.css

```diff
@@ -0,0 +1,51 @@
+/* The dropdown container */
+.dropdown {
+  float: left;
+  overflow: hidden;
+
+  a {
+    float: left;
+    color: white;
+    text-align: center;
+    text-decoration: none;
+    font-weight: 500;
+  }
+
+  .dropdownButton {
+    border: none;
```

Comment:
Can we add a cursor: pointer for on hover here? Not showing up locally for some reason even though it's a button. Don't know if it does for you

Reply:
Could you specify exactly? It seems to break on my end if I added that.

Reply:
Actually nvm, it doesn't make sense to do that since you can't click on the About text itself